ports/servoshell/egl/android/simpleservo.rs

```diff
@@ -30,7 +30,7 @@ use crate::egl::servo_glue::{
 use crate::prefs::{parse_command_line_arguments, ArgumentParsingResult};
 
 thread_local! {
-    pub static SERVO: RefCell<Option<ServoGlue>> = RefCell::new(None);
+    pub static SERVO: RefCell<Option<ServoGlue>> = const { RefCell::new(None) };
```

Comment:
Odd that clippy would complain about this, since this block cannot possibly be evaluated anytime other than at compile time, but sure.